src/screens/ChatNationsScreen/index.js

```diff
@@ -8,10 +9,37 @@ import { screen } from '../../global/Screens';
 import { resolveNation } from '../../utils/nations';
 import NavigatorComponent from '../../components/common/NavigatorComponent';
 import { startNationCreation } from '../../actions/modifyNation';
+import type { NationType } from '../../services/database/schemata';
 
+type Props = {
+  /**
+   * @desc React Native Navigation navigator object.
+   */
+  navigator: any,
```

Comment:
Please use Navigator type from `src/types/ReactNativeNavigation.js`. Feel free to add more fields/methods there if something is missing.